department-of-veterans-affairs/vets-api
# Pension - log confirmation id

## Summary

- *This work is behind a feature toggle (flipper): NO*
- We need to determine the benefit_intake_uuid, given a confirmation number. We previously logged the claim_id and the benefits_intake_uuid in PensionBenefitIntakeJob, but not confirmation_number. Logs the confirmation number in PensionBenefitIntakeJob.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74967

## Testing done

- Submit the form 21P-527EZ. Note the confirmation ID on the confirmation page. 
- View the logs, and confirm that the confirmation ID is logged along with the benefit intake ID

## Screenshots

<img width="527" alt="Screenshot 2024-01-30 at 4 24 17 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/7ec456f0-aae4-4b45-aa0f-2994be86557a">
<img width="868" alt="Screenshot 2024-01-30 at 4 24 35 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/5ead2e60-184b-4fed-a286-2e054be24e5b">
<img width="872" alt="Screenshot 2024-01-30 at 4 24 43 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/89efd463-f570-4296-a3af-b6f6669af144">
<img width="874" alt="Screenshot 2024-01-30 at 4 25 12 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/aeb6c0f5-d8e1-499e-8a3e-f1b0f1b3c404">

## What areas of the site does it impact?
Pension benefits

## Acceptance criteria
